src/yosys_mau/task_loop/logging.py

```diff
@@ -117,17 +120,17 @@ class LogContext:
     When set, the downgrading happens before the `LogEvent` is emitted.
     """
 
-    level: Level = "info"
-    """The minimum log level to display/log.
+    dest_levels: TaskContextDict[str, Level] = TaskContextDict()
+    """The minimum log level to display/log for named destinations.
```

Comment:
It's `dest_levels` here but `destination_label` below. It should be either consistently abbreviated or not abbreviated at all. Either choice works for me, but I'd probably use `destination`.